src/configuration/WiFiCredential.h

```diff
@@ -1,6 +1,6 @@
 /*
     SlimeVR Code is placed under the MIT license
-    Copyright (c) 2021 Eiren Rain & SlimeVR contributors
+    Copyright (c) 2022 TheDevMinerTV
```

Comment:
niet

Reply:
look at the file, git thought I copied it, but I didn't, I entirely rewrote it.

Reply:
Yeah but its still based on Eiren's copyright because the original was written by Eiren.

Reply:
Resolving things related to this, those can be brought up in a different PR. This PR review is lasting too long.

Resolve when you've read